Improve ConvertFile script and new checkPDFEncryptionAndValidity script (#39611)
demisto/content: Packs/CommonScripts/Scripts/CheckPDFEncryptionAndValidity/README.md

This script checks whether a given PDF file is valid and whether it is encrypted. The results are returned as fields in the file’s context.

> If an error occurs during execution, the script will not fail. Instead, the error details will be captured and returned in the file’s context.

Reviewer: ```suggestion
The script will not fail on invalid/ encrypted files. Instead, the error details will be captured and returned in the file’s context.
```
